src/documentation/pages/foundations-wip/accessibility/page-structure/examples/_meaningful-sequence.vue

```diff
@@ -0,0 +1,66 @@
+<template lang="pug">
+  div
+    <TitleAnchor title="Meaningful sequence" id="meaningful-sequence" titleSize="h2" />
+    .chi-grid
+      .chi-col.-w--12
+        .chi-breadcrumb
+          ol.-text--normal
+            li.chi-breadcrumb__item(v-for="({ url, text }) in breadcrumbs")
+              a(:href="url" rel="noopener" target="_blank") {{ text }}
+      .chi-col.-w--12
+        .-lh--3
+          .chi-badge.-dark.-outline.-xs.-mr--1(v-for="({ tooltipText, mainText }) in badges" :data-tooltip="tooltipText" data-position="bottom")
+            span {{ mainText }}
+      .chi-col.-w--12
+        p.-text
+          | <strong>Benefits:</strong> Blind keyboard user
+        p.-text
+          | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.
```

Comment:
@kosgor this needs to be updated with the info that there is in prod. There is some deviation. Update it acordingly